README.adoc

```diff
@@ -1103,8 +1103,7 @@ User.where.not('status = ? AND plan = ?', 'active', 'basic')
 
 === Redundant `all` [[redundant-all]]
 
-Using `all` as a receiver for Active Record query methods is redundant. 
-The result won't change without `all`, so it can be removed.
+Using `all` as a receiver is redundant. The result won't change without `all`, so it could be removed.
```

Comment:
While we’re here, do you think it’s worth aligning with RFC 2119 and using “should” here instead of a vague “could” (and my prior “can”)?

Reply:
Well, 'could' is a bit understated, so I don't think it's a problem to use 'should'.
I've addressed it with https://github.com/rubocop/rails-style-guide/pull/348/commits/9d5d5b145dd717125ec11e9d3dfb61c6ba978b5f.